### ISSUE TITLE
Screen Flickering

### ISSUE THREAD

**Issue by ilovefreesw** [arun-goud/Wintermelon#1]
My build here: https://github.com/ilovefreesw/Wintermelon

It works !! But when I log in back, the screen on the second monitor flickers for a second or two everytime. Does it happen at your end as well ?

**Comment by arun-goud** [comment]
It takes 3 seconds or so for my secondary monitor to return to its original state (duplicate --> extend) after the unlock operation. The transition on my side shows up as windows being shifted back to the secondary monitor and I don't see flickering on my second monitor.
Can you check if the flickering happens when you run the following in cmd terminal one by one?
```
C:\Windows\System32\DisplaySwitch.exe /clone
C:\Windows\System32\DisplaySwitch.exe /extend
C:\Windows\System32\DisplaySwitch.exe /internal
C:\Windows\System32\DisplaySwitch.exe /external
```
This does what pressing Win+P does.
If the flickering happens then you may have to update your display drivers.

**Comment by ilovefreesw** [comment]
When I run the task manually then it works but it doesn't work automatically. It happens on Windows 11, what you think can be a problem? Even when I run "Wintermelo.exe save" manually and lock the screen; it works. It's just not working from the Task Scheduler, even though the Task Scheduler shows that the task ran successfully.

**Comment by arun-goud** [comment]
I hope you have run the PowerShell script in admin mode as specified in the instructions. 
If it still doesn't work then have you tried playing with these settings in Task Scheduler? I'd recommend trying them one by one.
![image](https://user-images.githubusercontent.com/72409952/222315470-c32eb4e5-ba88-4b20-b252-59d2851a567e.png)

**Comment by ilovefreesw** [comment]
Yes, I did Admin PowerShell.  I will give it a shot and will report back.
